### DISCUSSION TITLE
There's a bizarre, seemingly AI-generated website posing as golangci-lint

### DISCUSSION THREAD

**Discussion by bqback** [golangci/golangci-lint#5078] [General]
The domain name is the same as the official https://golangci-lint.run/, but with `com` instead of `run`. Some of the content does seem to be on theme, but towards the end it devolves into completely vague LLM slop unrelated to golangci-lint.
It didn't show up on Google, but DuckDuckGo has it as the first result for `golangci-lint`.

My questions are:

- Why?
- Can anything be done about this?

**Comment by ldez** [comment]
This site is not related to golangci-lint.
This is a scam.

Sadly I don't know what we can do.

https://duckduckgo.com/?t=ffab&q=golangci-lint

You can report the result:

![Screenshot_20241018_023253](https://github.com/user-attachments/assets/4a0ce493-b55b-422f-bd3a-a4ed1e0d9a3f)

**Reply by ldez** [reply]
I don't know the "why".

The domain was bought on 2024-09-20 for one year (at least), and the site is a WordPress.

I guess it's for money but I don't know how they plan to do that :thinking:

**Comment by ldez** [comment]
The site is down.

**Reply by ldez** [reply]
The site is back...
